src/Servers/IIS/build/repo.props

```diff
@@ -16,24 +16,7 @@
   </PropertyGroup>
```

Comment:
You can delete this entire file. I think this is left over from ProjectRef work. It's not used anymore. Likewise for src/Servers/IIS/build/repo.targets. I think @jkotalik is cleaning that up right now in the 2.2 branch.